DueTimer.cpp

```diff
@@ -8,8 +8,6 @@
   Released into the public domain.
 */
 
-#include <Arduino.h>
-#if defined(_SAM3XA_)
```

Comment:
I guess I removed that because there is a similar check in the header file:
#ifdef __arm__

But I think that should be:
#if defined(_SAM3XA_)